products/jbrowse-web/src/tests/Loader.test.tsx

```diff
@@ -86,8 +101,6 @@ function FallbackComponent({ error }: FallbackProps) {
   return <div>there was an error: {String(error)}</div>
 }
 
-const initialTimestamp = 0
```

Comment:
What does getting rid of the timestamp for the tests do?

Reply:
i just made it an optional parameter since its only used for analytics